irohad/ametsuchi/block_query.hpp

```diff
@@ -18,6 +18,8 @@
 #ifndef IROHA_BLOCK_QUERY_HPP
 #define IROHA_BLOCK_QUERY_HPP
 
+#include <cmath>
```

Comment:
why is this here?

Reply:
Well, that's really rx-cpp issue and this include should be somewhere inside their code. This place is bad but other places are not really better. And yes, without this line some systems (at least, my one) get a compilation error about ambiguous call of std::abs somewhere inside rc-cpp internals.

Reply:
Same problem, anyway this include doesn't help

Reply:
Look at [this](https://gist.github.com/l4l/701b2d56dfc9b9714ab4955b1df16902) one